scripts/standalone.mk

```diff
@@ -204,22 +214,29 @@ RISCV_XCFLAGS += -DDHRY_ITERS=$(TARGET_DHRY_ITERS)
 endif
 
 ifeq ($(PROGRAM),coremark)
+ifeq ($(RISCV_SERIES),sifive-8-series)
 ifeq ($(RISCV_SERIES),sifive-7-series)
```

Comment:
This will never be true because `RISCV_SERIES` will never be both `sifive-8-series` and `sifive-7-series`

Reply:
Added a separate check for 8-series to use 7-series mtune for now, which can be changed in the future